youtube_dl/extractor/funimation.py

```diff
@@ -110,11 +110,24 @@ def _search_kane(name):
             headers = {}
             if self._TOKEN:
                 headers['Authorization'] = 'Token %s' % self._TOKEN
-            sources = self._download_json(
-                'https://www.funimation.com/api/showexperience/%s/' % video_id,
-                video_id, headers=headers, query={
-                    'pinst_id': ''.join([random.choice(string.digits + string.ascii_letters) for _ in range(8)]),
-                })['items']
+            try:
+                sources = self._download_json(
+                    'https://www.funimation.com/api/showexperience/%s/' % video_id,
+                    video_id, headers=headers, query={
+                        'pinst_id': ''.join([random.choice(string.digits + string.ascii_letters) for _ in range(8)]),
+                    })
+                sources = sources['items']
```

Comment:
Broke this out into a separate line so that `sources` is defined regardless of whether or not it contains a key `items`. This is useful because we can still grab the `errors` even if a `KeyError` is generated, which is what the code below does.